buildSrc/src/main/groovy/com/github/okbuilds/core/model/JavaLibTarget.groovy

```diff
@@ -22,6 +23,17 @@ class JavaLibTarget extends JavaTarget {
         } else {
             retrolambda = null
         }
+
+        // SqlDelight
+        if (project.plugins.hasPlugin('com.squareup.sqldelight')) {
+            File sqldelightDir = new File(project.projectDir, "build/generated/source/sqldelight")
+            if (!sqldelightDir.exists()) {
+                sqldelightDir.mkdirs()
+            }
+            sqldelight = new Scope(project, ["provided"] as Set, [sqldelightDir] as Set)
```

Comment:
this should be in android target if you want to specify provided

if it works for java libs as well, you need to support both provided and compileOnly configurations